### PR TITLE
Stats Widget: Adjust widget mobile layout

### PR DESCRIPTION
**Pull request by dognose24**
Related to #75107 partially

## Proposed Changes

* Use breakpoint `799px` in line with the dashboard to determine the mobile layout for the design of the Stats Widget.
* Adjust layout in line with the latest design except for the `7 Day Highlights` section.

|Before|After|
|-|-|
|截圖 2023-04-27 上午10 52 57|截圖 2023-04-27 上午11 29 04|

## Testing Instructions

* Open your local Jetpack site.
* Check out the branch of the Jetpack to the latest `trunk`.
* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Navigate to `/wp-admin/index.php`.
* Ensure the mobile layout of the Stats Widget looks reasonable.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: apps/odyssey-stats/src/widget/index.jsx [hunk 1]
```diff
@@ -42,8 +43,9 @@ export function init() {
 								target="__blank"
 								aria-label="Jetpack Stats Website"
 							>
-								<JetpackLogo size={ 25 } full />
+								<JetpackLogo size={ 25 } monochrome full />
```
Comment (kangzj) [inline apps/odyssey-stats/src/widget/index.jsx]:
```suggestion
								<JetpackLogo size={ 20 } monochrome full />
```

**Review by kangzj — APPROVED** [review]
Well done! It works well for me 👍 

I took the chance to make the logo in the footer smaller by the design hope you don't mind 😄

File: apps/odyssey-stats/src/widget/index.jsx (hunk 1 above)
Comment (dognose24) [reply apps/odyssey-stats/src/widget/index.jsx]: Nice catch! @kangzj

**Review by jsnmoon — APPROVED** [review]
Looks great!

**Review by kangzj — APPROVED** [review]

**Comment by kangzj** [comment]
I had some exploration before and want to dump my thoughts here for reference on how we further improve it.

The fact that the width of the widget container isn't directly related to the screen width because of the various breakpoints defined in `dashboard.css`, which means when the screen is wider the widget container is usually narrower than mobile layout. AFAIK, we have the following ways to tackle this.

1. Container Media Query: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries

2. Use normal Media Query, but we have to add quite a few rules

3. Define desktop (double columns) and mobile (single column) layouts and switch between the two with a callback from `ResizeObserver`, just like what's done in https://github.com/Automattic/wp-calypso/pull/76261

I would prefer the third solution, because
- Option 3 simple enough.
- Option 1 is not well supported among browsers.
- Option 2 would be over complicated.

Just for your reference, I'm happy if you go down other ways better 😄

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7966545

Thank you @dognose24 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7966545).

**Pull request merged**